### PR TITLE
add: `libspa-codec-bluez5-aac-bin`

### PR DESCRIPTION
**Pull request by Zahrun**
This adds AAC bluetooth codec to pipewire. It is not included in debian nor ubuntu due to non-free licensing https://wiki.debian.org/BluetoothUser/a2dp#AptX.2C_LDAC.2C_and_AAC_codecs_are_not_available_with_PulseAudio
The source for this binary is https://launchpadlibrarian.net/661532530/libspa-0.2-bluetooth_0.3.65-4~glasgall1_amd64.deb from https://launchpad.net/~aglasgall/+archive/ubuntu/pipewire-extra-bt-codecs.
However, if I put the deb in src, pacstall will try to install the deb. I just want to take one binary from this deb and put it in the right spot.
I am not sure if this is the best way to do it, please let me know.

### CODE REVIEW

File: packages/libspa-codec-bluez5-aac-bin/libspa-codec-bluez5-aac-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name="libspa-codec-bluez5-aac-bin"
+gives="libspa-codec-bluez5-aac"
+version="0.3.65-4~glasgall1"
+url="https://github.com/Zahrun/libspa-0.2-bluetooth/raw/main/libspa-0.2-bluetooth_${version}_amd64.deb."
```
Comment (Henryws) [inline packages/libspa-codec-bluez5-aac-bin/libspa-codec-bluez5-aac-bin.pacscript:4]:
```suggestion
url="https://github.com/Zahrun/libspa-0.2-bluetooth/raw/main/libspa-0.2-bluetooth_${version}_amd64.deb"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/libspa-codec-bluez5-aac-bin/libspa-codec-bluez5-aac-bin.pacscript (hunk 1 above)
Comment (Zahrun) [reply packages/libspa-codec-bluez5-aac-bin/libspa-codec-bluez5-aac-bin.pacscript:4]: If the extension is .deb, pacstall will install the deb file directly. The additionnal dot is intentional.

Comment (Henryws) [reply packages/libspa-codec-bluez5-aac-bin/libspa-codec-bluez5-aac-bin.pacscript:4]: Why can't the deb just be installed again?

Comment (Zahrun) [reply packages/libspa-codec-bluez5-aac-bin/libspa-codec-bluez5-aac-bin.pacscript:4]: It depends on stuff that are in the ppa libspa-0.2-modules (= 0.3.65-4~glasgall1) but 0.3.71-0~tux1 is to be installed
or in specific ubuntu versions libsbc1 (>= 2.0) but 1.5-3build2 is to be installed
Whereas just adding the one missing .so file at the right spot works just fine

Comment (Henryws) [reply packages/libspa-codec-bluez5-aac-bin/libspa-codec-bluez5-aac-bin.pacscript:4]: So can you not create a `tar.xz` with just the `.so` file in it?

Comment (Zahrun) [reply packages/libspa-codec-bluez5-aac-bin/libspa-codec-bluez5-aac-bin.pacscript:4]: I could, but then I have to do that every time there is a new version, whereas here I am just taking it out from the deb.
I’m fine to do the tar.xz if you prefer that.

Comment (Henryws) [reply packages/libspa-codec-bluez5-aac-bin/libspa-codec-bluez5-aac-bin.pacscript:4]: I'd like a `.tar.xz`.

Comment (Zahrun) [reply packages/libspa-codec-bluez5-aac-bin/libspa-codec-bluez5-aac-bin.pacscript]: Ok, although less automatic, this looks cleaner, I must admit

File: packages/libspa-codec-bluez5-aac-bin/libspa-codec-bluez5-aac-bin.pacscript [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
 depends=("libspa-0.2-bluetooth")
 incompatible=('debian:bullseye')
 
-prepare() {
-  dpkg-deb -x libspa-0.2-bluetooth_0.3.65-4~glasgall1_amd64.deb. .
-}
-
 install() {
-  sudo install -Dm644 ./usr/lib/x86_64-linux-gnu/spa-0.2/bluez5/libspa-codec-bluez5-aac.so "${pkgdir}/usr/lib/x86_64-linux-gnu/spa-0.2/bluez5/libspa-codec-bluez5-aac.so"
+  sudo install -Dm644 libspa-codec-bluez5-aac.so "${pkgdir}/usr/lib/x86_64-linux-gnu/spa-0.2/bluez5/libspa-codec-bluez5-aac.so"
```
Comment (Henryws) [inline packages/libspa-codec-bluez5-aac-bin/libspa-codec-bluez5-aac-bin.pacscript:12]:
```suggestion
  sudo install -Dm644 libspa-codec-bluez5-aac.so -t "${pkgdir}/usr/lib/x86_64-linux-gnu/spa-0.2/bluez5/"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**